application/src/main/java/net/corda/v5/application/uniqueness/model/UniquenessCheckStateRef.java

```diff
@@ -1,20 +1,23 @@
-package net.corda.v5.application.uniqueness.model
+package net.corda.v5.application.uniqueness.model;
 
-import net.corda.v5.base.annotations.CordaSerializable
-import net.corda.v5.crypto.SecureHash
+import net.corda.v5.base.annotations.CordaSerializable;
+import net.corda.v5.crypto.SecureHash;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Representation of a state reference. This type might also be attached to some of the error types
  * and returned through the client service. This representation does not depend on any specific ledger
  * model and is agnostic to both the message bus API and any DB schema that may be used to persist data
  * by the backing store.
- *
+ * <p>
  * Please note that this representation of a state ref is entirely different from the Ledger specific
- * [StateRef][net.corda.v5.ledger.utxo.StateRef] class. This class represents a state ref that the
+ * {@link net.corda.v5.ledger.utxo.StateRef} class. This class represents a state ref that the
  * uniqueness checker will process and may differ from the UTXO ledger data model.
  */
 @CordaSerializable
-interface UniquenessCheckStateRef {
-    val txHash: SecureHash
-    val stateIndex: Int
+public interface UniquenessCheckStateRef {
```

Comment:
Not for this PR, but @relyafi @nkovacsx should the uniqueness parts be in `ledger`? Especially since it mentions state ref (although not by class)

Reply:
Answers on a postcard, please :envelope:.